src/test/java/com/github/underscore/CollectionsTest.java

```diff
@@ -631,6 +631,16 @@ public boolean test(Integer item) {
         assertEquals("[2, 4, 6]", resultObject.toString());
     }
 
+    @Test
+    public void filterNotNull() {
+        final List<Integer> result = U.filterNotNull(asList(1, null, 3, 4, Integer.getInteger(null)));
```

Comment:
\[FindBugs\] WARNING: NP: Non\-virtual method call in com\.github\.underscore\.CollectionsTest\.filterNotNull\(\) passes null for nonnull parameter of Integer\.getInteger\(String\)